app/views/expenses/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<section class="devise-forms">
+  <h1>New expense</h1>
+
+  <%= render "form", expense: @expense %>
+
+  <br>
+
+  <div>
+    <%= link_to 'Back to Budget Categories', budget_categories_path %>
```

Comment:
- [ ] This back button should be **`<`** and should be on the top of the page based on the design.

- [ ] When I am on this new expense page and I click the back button I expect to be taken to the expense index page, not the categories page. Kindly fix that.

Reply:
Done. Thanks.